user/serializers.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+
+class RegisterSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["username", "password", "email"]
+        extra_kwargs = {"password": {"write_only": True}}
```

Comment:
While the password field is marked as write-only, it is important to ensure that passwords are hashed before being saved. The create method uses create_user, which handles password hashing correctly, so this is implemented properly.